eng/Directory.Build.Data.targets

```diff
@@ -125,6 +125,7 @@
     <MSBuild
       Projects="$(MSBuildThisFileDirectory)/ApiCompat/ApiCompat.csproj"
       Properties="TargetPackageName=$(PackageId);TargetOutputPath=$(IntermediateOutputPath)"
+      Targets="ApiCompatVerification"
```

Comment:
Why did adding this to VS require this update? Did VS try executing the default target?

Reply:
Yep